src/vs/workbench/contrib/notebook/browser/contrib/kernelDetection/notebookKernelDetection.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is this file add supposed to be a part of this PR?

Reply:
no, I did the wrong stash ;(